network_wrangler/transit/selection.py

```diff
@@ -380,10 +368,13 @@ def _filter_trips_by_timespan(
     # Filter freq to time that overlaps selection
     selected_freq_df = pd.DataFrame()
     for timespan in timespans:
-        selected_freq_df = selected_freq_df.append(
-            freq_df[
-                (freq_df["end_time"] >= timespan.start_time)
-                & (freq_df["start_time"] <= timespan.end_time)
+        selected_freq_df = pd.concat(
+            [
+                selected_freq_df,
+                freq_df[
+                    (freq_df["start_time"] >= timespan[0])
+                    & (freq_df["end_time"] <= timespan[1])
+                ],
             ]
```

Comment:
append method was removed starting from Pandas 2.0.0.
switch start and end time

Reply:
Start/end time switch was actually correct in original - b/c want to test if have ANY overlap.  
Append method fixed along with a few other bugs in: https://github.com/wsp-sag/network_wrangler/pull/327/commits/10f2dd47a7c47b17135d571c399851781a8fe5da